### ISSUE TITLE
Building nitter manually fails

### ISSUE THREAD

**Issue by tux93** [zedeus/nitter#452]
when trying tu build current master of nitter the build fails with:
```
/home/tux93/git/nitter/src/parser.nim(275, 9) Error: expected: ')', but got: 'keyword else'
Error: Build failed for package: nitter
... Execution failed with exit code 1
... Command: /usr/lib64/nim/bin/nim c --colors:on --noNimblePath -d:release -d:NimblePkgVersion=0.1.0 --path:'/home/tux93/.nimble/pkgs/asynctools-#pr_fix_for_latest' --path:/home/tux93/.nimble/pkgs/httpbeast-0.3.0 --path:'/home/tux93/.nimble/pkgs/asynctools-#pr_fix_compilation' --path:/home/tux93/.nimble/pkgs/jester-0.5.0 --path:/home/tux93/.nimble/pkgs/ws-0.4.4 --path:/home/tux93/.nimble/pkgs/dotenv-1.1.1 --path:/home/tux93/.nimble/pkgs/karax-1.2.1 --path:'/home/tux93/.nimble/pkgs/sass-#e683aa1' --path:/home/tux93/.nimble/pkgs/unicodedb-0.9.0 --path:/home/tux93/.nimble/pkgs/unicodedb-0.9.0 --path:/home/tux93/.nimble/pkgs/unicodedb-0.9.0 --path:/home/tux93/.nimble/pkgs/segmentation-0.1.0 --path:/home/tux93/.nimble/pkgs/unicodeplus-0.8.0 --path:'/home/tux93/.nimble/pkgs/regex-#2e32fdc' --path:/home/tux93/.nimble/pkgs/nimcrypto-0.5.4 --path:'/home/tux93/.nimble/pkgs/markdown-#abdbe5e' --path:'/home/tux93/.nimble/pkgs/packedjson-#d11d167' --path:'/home/tux93/.nimble/pkgs/supersnappy-#1.1.5' --path:'/home/tux93/.nimble/pkgs/redis-#94bcbf1' --path:'/home/tux93/.nimble/pkgs/redpool-#f880f49' --path:'/home/tux93/.nimble/pkgs/redis-#94bcbf1' --path:/home/tux93/.nimble/pkgs/sorta-0.2.0 --path:/home/tux93/.nimble/pkgs/grok-0.5.3 --path:/home/tux93/.nimble/pkgs/npeg-0.24.1 --path:/home/tux93/.nimble/pkgs/ups-0.0.7 --path:/home/tux93/.nimble/pkgs/sync-1.4.0 --path:/home/tux93/.nimble/pkgs/balls-3.6.0 --path:/home/tux93/.nimble/pkgs/criterion-0.2.8 --path:/home/tux93/.nimble/pkgs/grok-0.5.3 --path:/home/tux93/.nimble/pkgs/bytes2human-0.2.2 --path:/home/tux93/.nimble/pkgs/testes-0.10.0 --path:'/home/tux93/.nimble/pkgs/frosty-#0.3.1' -o:/home/tux93/git/nitter/nitter /home/tux93/git/nitter/src/nitter.nim
```

The offending line was introduced in cf47c1b8ad3affa43881ea661c8a5a11d206dfa0 and after reverting it I was able to build successfully

```
nim --version
Nim Compiler Version 1.2.12 [Linux: amd64]
```

**Comment by zedeus** [comment]
You're using an old version of Nim, please update and try again

**Comment by tux93** [comment]
this was addressed in f02515fda2ca51b545310b86581680227d8877b2
